schema/resolversTests/validation.test.js

```diff
@@ -260,5 +269,31 @@ describe("resolvers", () => {
       };
       expect(result).toMatchObject(expected);
     });
+
+    it("should be able to update latest date properties", async () => {
+      const answer = await createNewAnswer(firstPage, "Date");
+      const validation = await queryAnswerValidations(answer.id);
+      const result = await mutateValidationParameters({
+        id: validation.latestDate.id,
+        latestDateInput: {
+          custom: "2017-01-01",
+          offset: {
+            value: 8,
+            unit: "Months"
+          },
+          relativePosition: "After"
+        }
+      });
+      const expected = {
+        id: validation.latestDate.id,
+        customDate: "2017-01-01",
+        offset: {
+          value: 8,
+          unit: "Months"
+        },
+        relativePosition: "After"
+      };
+      expect(JSON.parse(JSON.stringify(result))).toMatchObject(expected);
```

Comment:
This can be simplified to: `expect(result).toMatchObject(expected);`

Reply:
This works if the test passes but if the objects don't match then you can get https://github.com/facebook/jest/issues/6730

This makes the error more readable when the test fails. I have added a comment and a tech debt ticket. https://trello.com/c/Ho4Rh8MH/333-resolver-tests-can-trigger-bug-in-jest